.github/workflows/ci-sql-cli.yaml

```diff
@@ -94,6 +94,7 @@ jobs:
     env:
       AIRFLOW__CORE__XCOM_BACKEND: astro.custom_backend.astro_custom_backend.AstroCustomXcomBackend
       AIRFLOW__ASTRO_SDK__STORE_DATA_LOCAL_DEV: True
+      _TYPER_FORCE_DISABLE_TERMINAL: 1
```

Comment:
This is also being used by typer for testing. See [here](https://github.com/tiangolo/typer/blob/master/scripts/test.sh#L8-L9).

Reply:
This seems to be required for tests which print the help output such as `test_usage` below.